src/test/java/uk/gov/ons/fsdr/tests/acceptance/steps/ServiceNowSteps.java

```diff
@@ -43,18 +43,18 @@ public void the_employee_is_correctly_create_in_ServiceNow_with(String employeeI
     String[] records = serviceNowMockUtils.getRecords();
     String create = records[records.length-1];
 
-    String lmName = "\"u_lm_first_name_2\":null,\"u_lm_last_name_2\":null";
+    String lmName = "\"u_line_manager\":\"null null\"";
 
     if(roleId.length() == FIELD_OFFICER_ROLE_ID_LENGTH) {
-      lmName = "\"u_lm_first_name_2\":\"Bob\",\"u_lm_last_name_2\":\"Jones\"";
+      lmName = "\"u_line_manager\":\"Bob Jones\"";
     }
     if(roleId.length() == COORDINATOR_ROLE_ID_LENGTH) {
-      lmName = "\"u_lm_first_name_2\":\"Dave\",\"u_lm_last_name_2\":\"Davis\"";
+      lmName = "\"u_line_manager\":\"Dave Davis\"";
     }
     String expectedMessageRootNode =
         "\"location\":\"London\",\"first_name\":\"Fransico\",\"last_name\":\"Buyo\",\"u_preferred_name\":null,\"u_badge_number\":null,"+lmName+",\"employee_number\":\""
             + roleId
-            + "\",\"u_job_role_2\":null,\"u_contract_start_date\":\"[0-9-]{10}\",\"u_contract_end_date\":\"[0-9-]{10}\",\"u_employment_status\":\"ACTIVE\",\"zip\":\"FA43 1AB\",\"u_ons_id\":\"fransico.buyo[0-9]{2}@domain\",\"u_ons_device_number\":null,\"home_phone\":null,\"mobile_phone\":\"0987654321\",\"active\":true,\"user_name\":\""+employeeId+"\"";
+            + "\",\"u_job_role_2\":null,\"u_contract_start_date\":\"[0-9-]{10}\",\"u_contract_end_date\":\"[0-9-]{10}\",\"u_employment_status\":\"ASSIGNED\",\"zip\":\"FA43 1AB\",\"u_ons_id\":\"fransico.buyo[0-9]{2}@domain\",\"u_ons_device_number\":null,\"home_phone\":null,\"mobile_phone\":\"0987654321\",\"active\":true,\"user_name\":\""+employeeId+"\"";
```

Comment:
I think there are a couple of tests where this value is "READY_TO_START" instead of "ASSIGNED", you should probably pass the expected value in from the feature file so that it will work for those tests as well